Makefile.am

```diff
@@ -1,3 +1,7 @@
-ACLOCAL_AMFLAGS = ${ACLOCAL_FLAGS} -Im4
+ACLOCAL_AMFLAGS = ${ACLOCAL_FLAGS} -I m4
 
+if WINDOWS
+SUBDIRS = xayagame
+else
 SUBDIRS = xayagame mover
+endif
```

Comment:
Why do you disable mover for Windows?  Does it not cross-compile yet (and if so, do you think it would be particularly hard to fix)?  Note that even if you have your own implementation in C#, there is nothing wrong with compiling the C++ version `moverd` as well on Windows.  And if possible, I would like to reduce the number of platform-conditional things to the absolute minimum.

Reply:
My target goal was to cross-compile core library only, not sure why anyone would need mover to run on windows. It had some compiling troubles, but yes, if you think its best to get rid of as much platform-conditional things as possible, I will get rid of those and cross-compile with mover.

Reply:
Yes, please try to compile the entire library for Windows.  The C++ mover could either be used by people directly who want to play around with it, or it could be used as starting point for someone who wants to write a game in C++ on Windows.  Both seem useful - so unless there are particular problems with cross-compiling it (which I don't expect), it seems simpler to just compile everything than add conditions.